### PR TITLE
Add JSON columns parsing with SQLite client

### PR DESCRIPTION
**Pull request by ricardogama**
This PR adds support for SQLite, parsing JSON columns on fetch.

Closes #16

### CODE REVIEW

File: test/postgres/index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+ * Module dependencies.
+ */
+
+import bookshelf from 'bookshelf';
+import jsonColumns from '../../src';
+import knex from 'knex';
+import knexfile from './knexfile';
+import should from 'should';
+import sinon from 'sinon';
+import { dropTable, recreateTable } from '../utils';
```
Comment (abelsoares) [inline test/postgres/index.js]: Sort.

File: test/sqlite/index.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+ * Module dependencies.
+ */
+
+import bookshelf from 'bookshelf';
+import jsonColumns from '../../src';
+import knex from 'knex';
+import knexfile from './knexfile';
+import should from 'should';
+import sinon from 'sinon';
+import { clearTable, dropTable, recreateTable } from '../utils';
```
Comment (abelsoares) [inline test/sqlite/index.js]: Sort.

File: test/postgres/index.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+  before(async () => {
+    await recreateTable(repository);
+  });
+
+  after(async () => {
+    await dropTable(repository);
+  });
+
+  describe('if a json column is not registered', () => {
```
Comment (abelsoares) [inline test/postgres/index.js]: How about `when a json ...`.

File: test/postgres/index.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+      Model.forge();
+
+      ModelPrototype.initialize.callCount.should.equal(1);
+
+      sinon.restore(ModelPrototype);
+    });
+  });
+
+  describe('if a json column is registered', () => {
```
Comment (abelsoares) [inline test/postgres/index.js]: Same as above.

File: test/sqlite/index.js [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+  afterEach(async () => {
+    await clearTable(repository);
+  });
+
+  after(async () => {
+    await dropTable(repository);
+  });
+
+  describe('if a json column is not registered', () => {
```
Comment (abelsoares) [inline test/sqlite/index.js]: Same as above.

File: test/sqlite/index.js [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+      Collection.forge({});
+
+      CollectionPrototype.initialize.callCount.should.equal(1);
+
+      sinon.restore(CollectionPrototype);
+    });
+  });
+
+  describe('if a json column is registered', () => {
```
Comment (abelsoares) [inline test/sqlite/index.js]: Same as above.

File: README.md [hunk 7]
```diff
@@ -52,7 +52,7 @@ bookshelf.Model.extend({
 
 ## Contributing
 
-Feel free to fork this repository and submit pull requests. To run the tests, duplicate the `test/knexfile.js.dist` file, update it to your needs and run:
+Feel free to fork this repository and submit pull requests. To run the tests, duplicate the *test/knexfile.js.dist* file, update it to your needs and run:
```
Comment (abelsoares) [inline README.md]: Add test information to a different section.

Comment (ricardogama) [inline README.md]: Please create a new PR for that.

**Pull request merged**